chia/rpc/full_node_rpc_client.py

```diff
@@ -184,6 +184,13 @@ async def get_block_records(self, start: int, end: int) -> List:
         # TODO: return block records
         return response["block_records"]
 
+    async def get_block_spends(self, header_hash: bytes32) -> List:
```

Comment:
If there is a chance of None, we should return Optional[List[CoinSpend]] otherwise List[CoinSpend]